src/main/resources/application.properties

```diff
@@ -1,3 +1,9 @@
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5432/my_ticket_service_db
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.jackson.serialization.fail-on-empty-beans=false
+
 connectionConfig.url = jdbc:postgresql://localhost:5432/my_ticket_service_db
 connectionConfig.username = postgres
 connectionConfig.password = postgres
```

Comment:
this is already redundant. remove it and class with custom db configuration

Reply:
It's about ConditionalOnProperty on DataSource, to check it's work

Reply:
got it. ok